core/src/main/java/org/testcontainers/utility/TestcontainersConfiguration.java

```diff
@@ -60,12 +60,12 @@
 
     @Deprecated
     public String getAmbassadorContainerImage() {
-        return (String) properties.getOrDefault("ambassador.container.image", "richnorth/ambassador:latest");
+        return String.valueOf(properties.getOrDefault("ambassador.container.image", "richnorth/ambassador:latest")).trim();
```

Comment:
Perhaps we should pre-process inputs in the constructor instead?

Reply:
That makes sense - moved that logic into constructor